### DISCUSSION TITLE
Avalonia is support font features (stylistics sets)?

### DISCUSSION THREAD

**Discussion by fembina** [AvaloniaUI/Avalonia#14606] [Q&A] [answered]
Avalonia is support font features (stylistics sets)?

In css for examples is: font-feature-settings: 'ss04' on, 'cv05' on;
In wpf is: Typography.[StylisticSet3](https://learn.microsoft.com/en-us/dotnet/api/system.windows.documents.typography.stylisticset1?view=windowsdesktop-8.0) to true

**Comment by Gillibald** [comment, marked as answer]
https://github.com/AvaloniaUI/Avalonia/pull/14157#pullrequestreview-1813019122
